Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialGameInstance.h"
+#include "GDKTestGymsGameInstance.generated.h"
+
+UCLASS()
+class GDKTESTGYMS_API UGDKTestGymsGameInstance : public USpatialGameInstance
+{
+	GENERATED_BODY()
+
+public:
+	virtual void Init() override;
+	bool Tick(float DeltaSeconds);
+
+private:
+	FTickerDelegate TickDelegate;
+	FDelegateHandle TickDelegateHandle;
+
+	float AverageFPS = 1.0f;
```

Comment:
Is 30/60 a better starting point?

Reply:
Maybe higher is better if were going to fail tests on < x fps, the bias is removed after 30 frames from the code above